src/components/common/PageLayout/styles.module.css

```diff
@@ -1,18 +1,11 @@
-.header {
-  position: fixed;
-  left: 0;
-  top: 0;
-  width: 100%;
-  z-index: 1201;
-}
```

Comment:
Why move this? The logic behind it was that PageLayout orchestrates the layout, and the header is only responsible itself, not its position on the page.